src/pages/layer-2-grants/index.tsx

```diff
@@ -0,0 +1,337 @@
+                    to $750,000 in total available.
+                  </PageText>
+
+                  <PageText mb={6} fontStyle='italic'>
+                    Proposals are due in 6 weeks from <strong>October 24th, 2022</strong>.
+                  </PageText>
+
+                  <PageText fontStyle='italic'>
+                    All of the details you’ll need to apply can be found below.
+                  </PageText>
+                </section>
+              </Stack>
+
+              <Stack spacing={10}>
+                <section id='introduction' ref={ref2}>
+                  <PageSection mb={6}>Introduction</PageSection>
+
+                  <Box mb={6}>
+                    <List>
+                      <ListItem>
+                        The Ethereum Foundation is launching a grants initiative to encourage
+                        research and development around layer 2 applications and education from
+                        researching L2 metrics, multi-layer block explorer, compression for rollups,
+                        and educating more users on L2.
+                      </ListItem>
+                      <ListItem>
+                        Layer 2 continues to grow rapidly, and the ecosystem needs more builders -
+                        Read below and apply (all genuine applications are reviewed and replied to
+                        with feedback).
+                      </ListItem>
+                    </List>
+                  </Box>
+                </section>
+              </Stack>
+
+              <Stack>
+                <section id='ideas' ref={ref3}>
+                  <PageSection mb={6}>Ideas</PageSection>
+
+                  <PageText mb={6} fontWeight='bold'>
+                    Surprise us with your creativity! But here are a few ideas to kickstart your
+                    imagination:
+                  </PageText>
+
+                  <Box mb={6}>
+                    <List>
+                      <ListItem>An open-source block explorer that supports all rollups</ListItem>
+                      <ListItem>Better compression for rollups</ListItem>
+                      <ListItem>
+                        Research and define metrics for better understanding and comparing L2
+                        activity -- e.g. transactions, TVL, etc
+                      </ListItem>
+                      <ListItem>
+                        Check out a full list{' '}
+                        <Link
+                          fontWeight={700}
+                          color='brand.orange.100'
+                          href='https://hackmd.io/@BcrNKqoOR1OZEFsNu7su9w/SkdvCP7zs'
+                          isExternal
+                          _hover={{ textDecoration: 'none' }}
+                        >
+                          here
+                        </Link>
+                      </ListItem>
+                      <ListItem>
```

Comment:
This looks like it could be a separate section separate from this list

Reply:
![Screen Shot 2022-10-24 at 10 23 36 AM](https://user-images.githubusercontent.com/15589226/197576590-b71f69f4-00c4-4b66-ae32-eabf5f1496c2.png)